Detector/DetFCChhCalDiscs/compact/Endcaps_coneCryo.xml

```diff
@@ -60,19 +60,19 @@
   </define>
   <readouts>
     <readout name="EMECPhiEta">
-      <segmentation type="FCCSWGridPhiEta" phi_bins="704" offset_phi="-pi+(pi/704.)" grid_size_eta="0.01" offset_eta="-2.515"/>
+      <segmentation type="FCCSWGridPhiEta" phi_bins="704" offset_phi="-pi+(pi/704.)" grid_size_eta="0.01" offset_eta="-2.51024"/>
```

Comment:
Why did these numbers change?

Reply:
It changed to be consistent with the [offset of barrel](https://github.com/HEP-FCC/FCCSW/blob/master/Detector/DetFCChhECalInclined/compact/FCChh_ECalBarrel_Common.xml#L56). Without this change it matters if we run reconstruction of barrel only, or with endcaps + forward (there is a shift once endcaps&forward are included).